src/PinchZoom/component.tsx

```diff
@@ -796,6 +796,11 @@ class PinchZoom extends React.Component<Props> {
   private _handlerOnTouchEnd = this._handlerIfEnable(
     (touchEndEvent: TouchEvent) => {
       this._fingers = touchEndEvent.touches.length;
+
+      if (isZoomInteraction(this._interaction) || (isDragInteraction(this._interaction) && this._zoomFactor !== 1)) {
```

Comment:
I worry about backward compatibility, can we add a new prop to enable this logic?

```tsx
if(props.isXXXXX){
  if (isZoomInteraction(this._interaction) || (isDragInteraction(this._interaction) && this._zoomFactor !== 1)) {
    cancelEvent(touchEndEvent);
  }
}
```